### ISSUE TITLE
UnboundLocalError: local variable 'lat_expanded' referenced before assignment

### ISSUE THREAD

**Issue by ShervanGharari** [ShervanGharari/EASYMORE#16]
This error comes up with easymore installation using `pip install easymore` from PyPI. it will be fixed in the next release.
It is fixed in the code on the repo (both branches). So please follow the local install and **_do not forget_** to uninstall the exiting installed easymore before installing it locally (`pip uninstall easymore`).
```
UnboundLocalError                         Traceback (most recent call last)
<ipython-input-33-2ad4158b6f4a> in <module>
----> 1 esmr.nc_remapper()
D:\PostDoc\EASYMORE-main\EASYMORE-main\easymore\easymore.py in nc_remapper(self)
     75             self.check_source_nc()
     76             # find the case
---> 77             self.NetCDF_SHP_lat_lon()
     78             # create the source shapefile for case 1 and 2 if shapefile is not provided
     79             if (self.case == 1 or self.case == 2)  and (self.source_shp == ''):
D:\PostDoc\EASYMORE-main\EASYMORE-main\easymore\easymore.py in NetCDF_SHP_lat_lon(self)
    596             self.lat = lat
    597             self.lon = lon
--> 598             self.lat_expanded = lat_expanded
    599             self.lon_expanded = lon_expanded
    600         # case #2 rotated lat/lon
UnboundLocalError: local variable 'lat_expanded' referenced before assignment
```

**Comment by ShervanGharari** [comment]
This is fixed in the develop branch for now. EASYMORE allows small tolerance for mesh of lat lon to create the source shapefile edges. please check with EMDNA and see if the shapefile is created for first last column and rows (close to -180, 180, 90 and +90)

**Comment by ShervanGharari** [comment]
is fixed in easymore version 0.0.2
